service/history/workflow/mutable_state_impl.go

```diff
@@ -3143,7 +3143,7 @@ func (ms *MutableStateImpl) ApplyBuildIdRedirect(
 	ms.GetExecutionInfo().BuildIdRedirectCounter = redirectCounter
 
 	// Re-scheduling pending workflow and activity tasks.
-	err = ms.reschedulePendingWorkflowTask(false)
+	err = ms.reschedulePendingWorkflowTask()
```

Comment:
these should be true but was scoped out from @stephanos PR based on https://github.com/temporalio/temporal/pull/8103/files#r2403627911

Reply:
Right; I didn't want to make that change since I was worried it would require more reviews and introduce more risk.